src/components/common/Label/styled.js

```diff
@@ -1,13 +1,13 @@
 import styled from 'styled-components';
+import '../../../styled';
```

Comment:
is this line needed?

Reply:
no I have removed that now. Left over by accident from when I was trying to get the themes working